### PR TITLE
mindspore增加论文案例视频搜索源

### PR DESCRIPTION
**Pull request by 2511689622**

### CODE REVIEW

**Comment by shishupei** [comment]
开始门禁检查，等门禁检查通过后可以合入

**Comment by shishupei** [comment]
<table style="border-collapse: collapse" border=1>
    <tr>
        <th>检查项</th>
        <th>状态</th>
        <th>日志</th>
    </tr>

<tr>
    <td>开源代码漏洞扫描</td>
    <td>&#10060;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySearch-Import/8/8_开源代码漏洞扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>敏感信息扫描</td>
    <td>&#9989;</td>
    <td></td>
</tr>

<tr>
    <td>Check代码检查</td>
    <td>&#10060;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/EasySearch-Import/8/8_Check代码检查.html">查看日志</a></td>
</tr>

<tr>
    <td>流水线链接</td>
    <td colspan="2"><a href="https://devcloud.cn-north-4.huaweicloud.com/cicd/project/3a76c1785dda4b13a399937d1978f240/pipeline/detail/0dfc2b00b34c48868ac855f62a1586aa/c640880a7c0241f8819314238609b308">点击跳转</a></td>
</tr>
</table>

**Comment by shishupei** [comment]
开始门禁检查，等门禁检查通过后可以合入

**Comment by shishupei** [comment]
<table style="border-collapse: collapse" border=1>
    <tr>
        <th>检查项</th>
        <th>状态</th>
        <th>日志</th>
    </tr>

<tr>
    <td>敏感信息扫描</td>
    <td>&#9989;</td>
    <td></td>
</tr>

<tr>
    <td>流水线链接</td>
    <td colspan="2"><a href="https://devcloud.cn-north-4.huaweicloud.com/cicd/project/3a76c1785dda4b13a399937d1978f240/pipeline/detail/0dfc2b00b34c48868ac855f62a1586aa/f029fe3152fb43fdab0408e9889e3197">点击跳转</a></td>
</tr>
</table>

File: mindspore/src/main/java/App.java [hunk 1]
```diff
@@ -6,6 +6,12 @@
 public class App {
     private static final String TARGET = System.getenv("TARGET");
 
+    private static final String TARGET_PAPERS = System.getenv("TARGET") + "/papers";
+
+    private static final String TARGET_CASES = System.getenv("TARGET") + "/cases";
+
+    private static final String TARGET_COURSES = System.getenv("TARGET") + "/courses";
+
     private static final String APPLICATION_PATH = System.getenv("APPLICATION_PATH");
```
Comment (zhongjun2) [inline mindspore/src/main/java/App.java:16]: lgtm

**Review by zhongjun2 — APPROVED** [review]
approve

**Pull request merged**